tests/cql/CqlIntervalOperatorsTest.xml

```diff
@@ -1422,12 +1424,10 @@
 			<output>5.0'g'</output>
 		</test>
 		<test name="DateTimeWidth">
-			<expression>width of Interval[DateTime(2012, 1, 5), DateTime(2012, 1, 25)]</expression>
-			<output>20 days</output>
+			<expression invalid="true">width of Interval[DateTime(2012, 1, 5), DateTime(2012, 1, 25)]</expression>
 		</test>
 		<test name="TimeWidth">
-			<expression>width of Interval[@T05:59:59.999, @T15:59:59.999]</expression>
-			<output>36000000 milliseconds</output>
+			<expression invalid="true">width of Interval[@T05:59:59.999, @T15:59:59.999]</expression>
 		</test>
```

Comment:
[Appendix B 9.31 Width](http://build.fhir.org/ig/HL7/cql/09-b-cqlreference.html#width) says:
> Note that because CQL defines duration and difference operations for date and time valued intervals, width is not defined for intervals of these types.